### PR TITLE
some ui changes

### PR DESCRIPTION
**Pull request by YoniX44**
hellow from better pull request

### CODE REVIEW

File: src/Containers/Signin/Signin.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const Signin = () => {
       <Typography className={classes.title}>
         ברוכים הבאים לפומדרו התחברו לאתר כדי להמשיך
       </Typography>
+      <Typography> כמה כיף להיות בפומדרו!! 🍕🍕🍕 </Typography>
```
Comment (idobarlev) [inline src/Containers/Signin/Signin.js:18]: NOT GOOD

File: src/Containers/Signin/Signin.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const Signin = () => {
       <Typography className={classes.title}>
         ברוכים הבאים לפומדרו התחברו לאתר כדי להמשיך
       </Typography>
```
Comment (idobarlev) [inline src/Containers/Signin/Signin.js:17]: More not good

File: src/Containers/Signin/Signin.style.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ export const useStyles = makeStyles((theme) => ({
     fontFamily: "Rubik",
 
     "& :hover": {
-      color: "red",
+      fontWeight: "bold",
```
Comment (idobarlev) [inline src/Containers/Signin/Signin.style.js:15]: Change to blue

**Review by idobarlev — CHANGES_REQUESTED** [review]
Check the feedback!

File: src/Containers/Signin/Signin.js (hunk 2 above)
Comment (YoniX44) [reply src/Containers/Signin/Signin.js:17]: yo son of a gun

File: src/Containers/Signin/Signin.js (hunk 1 above)
Comment (YoniX44) [reply src/Containers/Signin/Signin.js:18]: :(

**Review by idobarlev — APPROVED** [review]

**Pull request merged**